File: 01.Java-Basic/03.ConditionalStatementsLab/src/AreaOfFigures.java
import java.util.Scanner;

public class AreaOfFigures {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String figure = scanner.nextLine();
        double p1 = Double.parseDouble(scanner.nextLine());

        double area = 0;
        if (figure.equals("square")) {
            area = p1 * p1;
        } else if (figure.equals("rectangle")) {
            double p2 = Double.parseDouble(scanner.nextLine());
            area = p1 * p2;
        } else if (figure.equals("circle")) {
            area = Math.PI * Math.pow(p1, 2);
        } else if (figure.equals("triangle")) {
            double p2 = Double.parseDouble(scanner.nextLine());
            area = (p1 * p2) / 2;
        }

        System.out.printf("%.3f", area);
    }
}

File: JavaScript/01.JavaScriptBasic/03Conditional-Statements-Lab/src/AreaOfFigures.js
function figureArea(input) {
    let figure = input[0];
    let area = undefined;
    switch (figure) {
        case "square":
            let side = Number(input[1]);
            area = side * side;
            break;

        case "rectangle":
            let a = Number(input[1]);
            let b = Number(input[2]);
            area = a * b;
            break;

        case "circle":
            let radius = Number(input[1]);
            area = Math.PI * Math.pow(radius, 2);
            break;

        case "triangle":
            let base = Number(input[1]);
            let height = Number(input[2]);
            area = (base * height) / 2;
            break;
    
        default:
            break;
    }

    console.log(area.toFixed(3));
}

figureArea(["square", "5"]);